### ISSUE TITLE
batch-transcode-video deletes successful result

### ISSUE THREAD

**Issue by JMoVS** [nwronski/batch-transcode-video#17]
Hi,

I just set up a new machine (Microsoft Surface Book) with this and now successful encodes end with a non-descriptive error and the output file gets deleted. The funny thing is: It works at version 1.2.0 of batch-transcode-video on a same hardware Surface Book.

The main difference is the one where it works runs Ubuntu 16.04 with batch-transcode-video 1.2.0 and the one where it doesn't runs Ubuntu 18.04 and doesn't work with either 1.2.0 or 1.3.0

Any ideas what that might be? Is it an artifact of https://github.com/nwronski/batch-transcode-video/issues/10#issuecomment-546709841 which would be solved by https://github.com/nwronski/batch-transcode-video/issues/10#issuecomment-465666887

**Comment by JMoVS** [comment]
works:

- Windows 10 Subsystem for Linux Ubuntu 16.04.6 LTS
- nodejs v4.2.6
- npm v3.5.2
- batch-transcode-video@1.2.0

doesn't work:
- Windows 10 Subsystem for Linux Ubuntu 18.04.3 LTS
- nodejs v8.10.0
- npm v3.5.2
- batch-transcode-video@1.2.0 and batch-transcode-video@1.3.0

**Comment by JMoVS** [comment]
```
 - ERRORED:  Lorna Doone.mkv
Error: [TranscodeError: Transcode probably did not succeed for file.]
```

**Comment by JMoVS** [comment]
no more info given - I have no idea what's causing this...
output file is definitely not existing

**Comment by JMoVS** [comment]
oh kewl! Are you slicing a new release with updated dependencies and all the shenanigans now?

**Comment by nwronski** [comment]
yes, just published [v2.0.0](https://github.com/nwronski/batch-transcode-video/releases/tag/v2.0.0). let me know if that does the trick, thanks!

```
npm i -g batch-transcode-video@2
```

**Comment by JMoVS** [comment]
Just checked, it worked!
